text/3151-scoped-threads.md

```diff
@@ -0,0 +1,237 @@
+- Feature Name: scoped_threads
+- Start Date: 2019-02-26
+- RFC PR: [rust-lang/rfcs#3151](https://github.com/rust-lang/rfcs/pull/3151)
+- Rust Issue: [rust-lang/rust#93203](https://github.com/rust-lang/rust/issues/93203)
+- Translators: [[@FizzyElt](https://github.com/FizzyElt)]
+- Commit: [The commit link this page based on](https://github.com/rust-lang/rfcs/blob/6de9eb35193852dd910c38f5c2c41e6f3e2a8ed4/text/3151-scoped-threads.md)
+- Updated: 2022-07-05
+
+# 概要
+[概要]: #summary
+
+在標準函式庫中新增作用域執行緒，其允許借用父執行續的變數來產生執行緒。
```

Comment:
```suggestion
在標準函式庫中新增作用域執行緒，其允許產生可借用父執行緒之變數的執行緒。
```

Reply:
Sorry but your new change is still not quite right. We are not borrowing variables to spawn a new thread. We are spawning threads that allow to borrow variables.